src/transport/Transport.java

```diff
@@ -56,6 +56,9 @@ public T getDriver() {
 
     public abstract void finishMoving();
 
+    public void printType() {
+    }
```

Comment:
```suggestion
    public abstract void printType();
```
абстрактный метод не имеет тела, см. выше методы, там точно так же, не знаю, почему у тебя здесь возникли трудности, если выше всё то же самое ты смог написать)